uwsgi/admin.wsgi

```diff
@@ -66,7 +66,7 @@ if not os.environ.get("LOCALDEV"):
         log.critical("GOOGLE_APPLICATION_CREDENTIALS provided, but does not exist")
         sys.exit(1)
 
-if os.environ.get("LOCALDEV") and "GOOGLE_APPLICATION_CREDENTIALS" not in os.environ:
+if os.environ.get("LOCALDEV") and ("GOOGLE_APPLICATION_CREDENTIALS" not in os.environ or not os.path.exists(os.environ.get("GOOGLE_APPLICATION_CREDENTIALS"))):
```

Comment:
Nit: This can be simplified by:

```suggestion
if os.environ.get("LOCALDEV") and not os.path.exists(os.environ.get("GOOGLE_APPLICATION_CREDENTIALS", ""))):
```

I think.